packages/admin-ui/src/common/types.ts

```diff
@@ -894,6 +894,12 @@ export interface PackageVersionData {
   commit?: string;
 }
 
+export interface TopicDataItem {
```

Comment:
Can you find a better name for this type? Topic Data and Item are very 3 generic works that are still generic together. The name should be scoped to the usage of this type